src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProvider.java

```diff
@@ -342,7 +344,7 @@ private VirtualMachine cloneVM(VirtualMachine vm, String newVMname, Instance ins
             Task task = vm.cloneVM_Task(vmFolder, newVMname, vmcs);
 
             String result = task.waitForTask();
-            if (Task.SUCCESS != result) {
+            if (!Task.SUCCESS.equals(result)) {
```

Comment:
be careful about the type of Task.SUCCESS,  is it an enum or String type ? if it is enum, they will never be equals